pmm/aws-staging-start.groovy

```diff
@@ -81,6 +81,12 @@ pipeline {
             choices: ['perfschema', 'slowlog'],
             description: "Query Source for Monitoring",
             name: 'QUERY_SOURCE')
+        string(
+            defaultValue: '',
```

Comment:
Can we have a default value here, something like "latest" or "dev-latest"?

Reply:
no, because usually we don't want to run version service next to PMM Server.